contracts/lib/math/SignedFixedPoint.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "./LogExpMath.sol";
+import "../helpers/BalancerErrors.sol";
+
+/* solhint-disable private-vars-leading-underscore */
+
+library SignedFixedPoint {
```

Comment:
Wouldn't be possible to support both signed and unsigned ops in the same `FixedPoint` lib? Similarly to how we are doing in `Math`

Reply:
Yes, sorry - we're actually not using this file at all anymore, I forgot to delete it.